main.tf

```diff
@@ -19,7 +21,7 @@ module "service_account_label" {
 resource "aws_iam_role" "service_account" {
   for_each           = toset(compact([module.service_account_label.id]))
   name               = each.value
-  description        = "Role assumed by Kubernetes ServiceAccount ${var.service_account_namespace}:${var.service_account_name}"
+  description        = "Role assumed by Kubernetes ServiceAccount ${var.service_account_namespace}:${local.service_account_scope}"
```

Comment:
use `format()/join()` to avoid deprecation warnings

Reply:
@SweetOps what deprecation warnings? 0.11 deprecation warnings? Or am I missing something?